{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1739674120610, "stop": 1739674120610}], "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "'secret_sauc'"}, {"name": "driver", "value": "'Edge'"}], "start": 1739674120403, "stop": 1739674120646, "uuid": "740e1072-10d5-432d-9747-82a7b44af921", "historyId": "39802564165ab5aad57da0e625718d99", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1335-749"}, {"name": "thread", "value": "1939-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}